src/Microsoft.AspNetCore.DeveloperCertificates.Tools/Program.cs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Microsoft.AspNetCore.Certificates.Generation;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.DeveloperCertificates.Tools
+{
+    class Program
+    {
+        private const int CriticalError = -1;
+        private const int Success = 0;
+        private const int ErrorCreatingTheCertificate = 1;
+        private const int ErrorSavingTheCertificate = 2;
+        private const int ErrorExportingTheCertificate = 3;
+        private const int ErrorTrustingTheCertificate = 4;
+
+        public static readonly TimeSpan HttpsCertificateValidity = TimeSpan.FromDays(365);
+
+        public static int Main(string[] args)
+        {
+            try
+            {
+                var app = new CommandLineApplication
+                {
+                    Name = "dotnet-developercertificates"
+                };
+
+                app.Command("https", c =>
+                {
+                    var exportPath = c.Option("-ep|--export-path",
+                        "Full path to the exported certificate",
+                        CommandOptionType.SingleValue);
+
+                    var password = c.Option("-p|--password",
+                        "Password to use when exporting the certificate with the private key into a pfx file",
+                        CommandOptionType.SingleValue);
+
+                    CommandOption trust = null;
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                    {
+                        trust = c.Option("-t|--trust",
+                            "Trust the certificate on the current platform",
+                            CommandOptionType.NoValue);
+                    }
+
+                    c.HelpOption("-h|--help");
+
+                    c.OnExecute(() => EnsureHttpsCertificate(exportPath, password, trust));
+                });
+
+                app.HelpOption("-h|--help");
+
+                app.OnExecute(() =>
+                    {
+                        app.ShowHelp();
+                        return Success;
+                    });
+
+                return app.Execute(args);
+            }
+            catch
```

Comment:
Log the error. Also, consider using `IReporter`.

Example: https://github.com/aspnet/DotNetTools/blob/c4788107f50d58bc730a420516f9f62505159652/src/Microsoft.Extensions.SecretManager.Tools/Program.cs#L39-L46